man/fi_getinfo.3.md

```diff
@@ -581,10 +581,17 @@ fabric.  See `fi_av`(3).
   the string is address and/or provider specific, but in general follows
   a URI model:
 
-  address_format[://[node][:[service][/[field3]...]]]
+  address_format[://[node][:[service][?[key=value][&key2=value2]...]]]
```

Comment:
Should probably keep `field3` in here.  We're not disallowing characters after `/` and before `?`.  We're only suggesting that any `key=value` tuples go after `?` and be delimited by `&`.